test/models/report_test.rb

```diff
@@ -3,7 +3,18 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'user can not edit report' do
+    report = reports(:first)
```

Comment:
```suggestion
    report = reports(:report_by_alice)
```

Fixtureの定義を見に行かなくても中身が想像できる名前の方がベターです